source/languages/en/riakcs/cookbooks/configuration/Configuring-Riak-CS.md

```diff
@@ -291,33 +307,30 @@ Instructions on creating your own CA cert can be found
 
 Riak CS can interact with S3 clients in one of two ways:
 
-* A **proxy** configuration enables an S3 client to communicate with
-  Riak CS as if it were Amazon S3 itself, i.e. using typical Amazon URLs.
-* A **direct** configuration requires that an S3 client connecting to
-  Riak CS be configured for an "S3-compatible service," i.e. with a Riak
-  CS endpoint that is not masquerading as Amazon S3. Examples of such
-  services include [Transmit](http://panic.com/transmit/),
-  [s3cmd](http://s3tools.org/s3cmd), and
+* A **proxy** configuration enables an S3 client to communicate with Riak CS as
```

Comment:
Suggest either linking to this (http://basho.com/riak-cs-proxy-vs-direct-configuration/) or making sure all the content is here (this may be a redundant comment, can't be bothered to view the full doc given the terrible performance I'm experiencing at the moment)